testing/yubico-piv-tool/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor: Christophe BERAUD-DUFOUR <christophe.berauddufour@gmail.com>
+# Maintainer:
+pkgname=yubico-piv-tool
+pkgver=1.4.3
+pkgrel=0
+pkgdesc="PIV Tools for yubikey on RaspberryPi"
+url="https://developers.yubico.com/yubico-piv-tool"
+arch="all"
+license="BSD"
+depends="openssl acf-openssl libcrypto1.0 libressl2.5-libcrypto pcsc-lite ccid"
```

Comment:
I think none of the dependencies are required to be stated here. `abuild` will be pick the .so library dependencies automatically for you.